app/(auth)/_layout.tsx

```diff
@@ -0,0 +1,11 @@
+import { Slot } from 'expo-router';
+
+export default function AuthLayout() {
+  return (
+    <Slot
+      screenOptions={{
+        unmountOnBlur: true,
+      }}
+    />
+  );
+}
```

Comment:
This is an interesting pattern, let's add a link to the documentation and a short note about what it does

Reply:
bd97d6bd5eba88e7e416336a07dfa0aad12ae138

Reply:
Style: verb at the beginning of the comment should be "Render", not "Renders"

Reply:
1. 2108e254c86b9ec0b984ef0d5e515c1bf097593b
2. We may not need it, just added it because I saw it on other layout on the app. Nothing about it in expo docs

Reply:
Ok, why do we have this layout at all? Does it do anything? Can we remove the file completely?

Reply:
We need the setup this way so that we can prevent the rendering of the page header title and back button

<img width="453" alt="Screenshot 2024-11-03 at 20 35 34" src="https://github.com/user-attachments/assets/e1e4ca57-c49b-4264-8a07-5898c004560d">

Reply:
Oh ok, then let's leave it

We added a comment about that here:

https://github.com/upleveled/expo-example-fall-2024-atvie/pull/1/files#diff-2266465565dd371a3e61d7091960e7bb8993f67124d6e21140fcea31d4740db3